package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "gulp docs-generate && webpack-dev-server --inline --hot --content-base src/docs/",
     "docs-upload-gh": "scripts/docs-upload-gh.sh",
     "docs-upload": "gulp docs-upload",
-    "jest": "NODE_ENV=test TZ=UTC jest",
+    "jest": "NODE_ENV=test TZ=UTC jest --runInBand",
```

Comment:
For some very strange reason, this flag was needed otherwise the tests would hang on travis. :man_shrugging: